### PR TITLE
vkoliesov_project - add models of Product

### PR DESCRIPTION
**Pull request by vkoliesov**
Sorry for delay

### CODE REVIEW

File: gastronom/product/models.py [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+    descriptions = models.CharField(max_length=500)
+    raiting = models.DecimalField(max_digits=3, decimal_places=1)
+    
+    def __str__(self):
+        return self.name
+
+
+class Media(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name='mediafiles')
+    photo = models.ImageField(blank=True)
```
Comment (oshevelo) [inline gastronom/product/models.py:14]: add path

File: gastronom/product/models.py [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+class Characteristic(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name='characteristics')
+    characteristic = models.CharField(max_length=20)
+    descriptions = models.CharField(max_length=500)
+    
+    def __str__(self):
+        return self.characteristic
+
+    
+class Feedback(models.Model):
```
Comment (oshevelo) [inline gastronom/product/models.py:26]: delete this

File: gastronom/product/models.py [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+    question_text = models.CharField(max_length=500)
+    pub_date = models.DateTimeField('date published')
+    likes = models.IntegerField(default=0)
+    dislikes = models.IntegerField(default=0)
+    
+    def __str__(self):
+        return self.question_text
+    
+    
+class Answer(models.Model):
```
Comment (vkoliesov) [inline gastronom/product/models.py:53]: delete

Comment (vkoliesov) [inline gastronom/product/models.py:53]: delete

File: gastronom/product/models.py (hunk 2 above)
Comment (vkoliesov) [inline gastronom/product/models.py:26]: delete

File: gastronom/product/models.py [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+    disadvantuges = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+    likes = models.IntegerField(default=0)
+    dislikes = models.IntegerField(default=0)
+    
+    def __str__(self):
+        return self.feedback_text
+
+
+class Question(models.Model):
```
Comment (vkoliesov) [inline gastronom/product/models.py:41]: delete

**Pull request closed without merging**